### DISCUSSION TITLE
attributes via hx:...

### DISCUSSION THREAD

**Discussion by sgruendel** [wimdeblauwe/htmx-spring-boot#93] [General]
With Thymeleaf 3.x, you can use any attribute and run it through Thymeleaf's default attribute processor: https://www.thymeleaf.org/doc/tutorials/3.1/usingthymeleaf.html#setting-the-value-of-any-attribute-default-attribute-processor

So what's the benefit of "hx:get" over "th:hx-get", or is it just relevant when using Thymeleaf 2.x?

**Comment by wimdeblauwe** [comment]
There is no benefit other then being just a little bit shorter. To be honest, I did not know that `th:hx-get` would just work before the library was written. So now that we have it anyway, we might as well keep it.
